custom-namespaces.yaml

```
@@ -338,6 +338,11 @@ firefox_accounts:
     - loines@mozilla.com
   pretty_name: Firefox Accounts
   views:
+    fxa_all_events:
```

The existing convention seems to be that the table views have an `fxa_` prefix while the events views (like this one) don't.  So if we want to continue that convention, this would be:
```suggestion
    all_events:
```
(with corresponding change to the explore's `base_view` property)